src/providers/wms/qgswmsdataitems.cpp

```diff
@@ -365,6 +365,21 @@ QString QgsWMSLayerItem::createUri()
   QString style = !mLayerProperty.style.isEmpty() ? mLayerProperty.style.at( 0 ).name : QString();
   mDataSourceUri.setParam( QStringLiteral( "styles" ), style );
 
+  // Check for layer dimensions
+  if ( !mLayerProperty.dimensions.empty() )
```

Comment:
No need for this check. The loop will be run 0 times if it's not empty.

Reply:
Yes, I see.